controllers/customer_controller_test.go

```diff
@@ -0,0 +1,49 @@
+package controllers
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/stakater/jira-service-desk-operator/api/v1alpha1"
+	mockData "github.com/stakater/jira-service-desk-operator/mock"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Customer Controller", func() {
+
```

Comment:
Ideally this should have a `beforeAll` block to create a project and `afterAll` block to delete that project